frontend/src/pages/projects/projectSettings/ModelBiasSettingsCard.tsx

```diff
@@ -52,7 +52,7 @@ const ModelBiasSettingsCard: React.FC<ModelBiasSettingsCardProps> = ({ namespace
           isLiveRegion
           isInline
         >
-          {error?.message}
```

Comment:
You need to check that the error code is a 404 before displaying this message - if the error code is anything else you should display the message as the service would have failed for a different reason.